doc/docker.md

```diff
@@ -0,0 +1,64 @@
+[<- Back to Index](index.md)
+
+# Docker Image
+
+Yet Analytics publishes Docker container images of SQL LRS on [DockerHub](https://hub.docker.com/repository/docker/yetanalytics/lrsql) in the format `yetanalytics/lrsql:<release version | latest>`.
+
+## Usage
+
+You can run SQL LRS directly from the Docker CLI in its default SQLite configuration:
+
+``` shell
+docker run \
+    -it \
+    -p 8080:8080 \
+    -e LRSQL_API_KEY_DEFAULT=my_key \
+    -e LRSQL_API_SECRET_DEFAULT=my_secret \
+    -e LRSQL_ADMIN_USER_DEFAULT=my_username \
+    -e LRSQL_ADMIN_PASS_DEFAULT=my_password \
+    yetanalytics/lrsql:latest
+```
+
+Note that the `-it` option will give you a pseudo-TTY and attach you to the container, allowing you to stop the SQL LRS container with ^C. It is not needed for production use, where `-d` would be preferable. See the [docker run docs](https://docs.docker.com/engine/reference/commandline/run/) for more information.
+
+See [Configuration Variables](env_vars.md) for more options.
+
+### Other DBMSs
+
+The SQL LRS Dockerfile uses the SQLite run script in `bin` as the default `CMD`. To change to another DBMS use the corresponding script. For instance to use PostgreSQL:
+
+``` shell
+docker run \
+    -it \
+    -p 8080:8080 \
+    -e LRSQL_API_KEY_DEFAULT=my_key \
+    -e LRSQL_API_SECRET_DEFAULT=my_secret \
+    -e LRSQL_ADMIN_USER_DEFAULT=my_username \
+    -e LRSQL_ADMIN_PASS_DEFAULT=my_password \
+    -e LRSQL_DB_HOST=0.0.0.0 \
+    -e LRSQL_DB_PORT=5432 \
+    -e LRSQL_DB_NAME=lrsql_db \
+    -e LRSQL_DB_USER=lrsql_user \
+    -e LRSQL_DB_PASSWORD=my_password \
+    -e LRSQL_DB_SCHEMA=lrsql \
+    yetanalytics/lrsql:latest \
+    /lrsql/bin/run_postgres.sh
+```
```

Comment:
Don't you need to setup a running Postgres instance on the image first? I got a `java.net.ConnectException: Connection refused` when I copy-pasted this code in, for obvious reasons. Again, an explanation probably isn't needed for users who know what they're doing, but I definitely think in this case you should show how to prepare the Docker container for PG.

Reply:
Yep, I feel it might be out of scope to provide the actual step-by-step on that, but a note that you'll need to set up Postgresql first would be good. I leave that up to @cliffcaseyyet actually, if you want a full config then I'll go ahead and do that `compose.yml` file I mentioned.

Reply:
just the note, pending what Cliff wants https://github.com/yetanalytics/lrsql/pull/169/commits/ea1a7c4f5d42cb87b3fc3222383834f0c903abd8

Reply:
@kelvinqian00 see https://github.com/yetanalytics/lrsql/pull/169/commits/31519d88b723401886c0e4593e63dfa6e8d991d6